agent/agentscripts/agentscripts.go

```diff
@@ -314,6 +320,15 @@ func (r *Runner) run(ctx context.Context, script codersdk.WorkspaceAgentScript)
 		} else {
 			logger.Info(ctx, fmt.Sprintf("%s script completed", logPath), slog.F("execution_time", execTime), slog.F("exit_code", exitCode))
 		}
+
+		_, err = r.scriptCompleted(ctx, &proto.WorkspaceAgentScriptCompletedRequest{
```

Comment:
Is this error handled?

Reply:
Oops, my bad. As this runs in a defer block what is the best way to handle this error? Should we just log that it failed?

Reply:
Probably just log, yeah; I don't think there's much else we can do.